providers/meteoblue.go

```diff
@@ -96,4 +104,21 @@ func (p *MeteoBlueProvider) FetchData(qp map[string]string) (*plumber.BaseData,
 	return baseData, nil
 }
 
-// Maybe add a MeteoBlueQueryParams like OpenMeteoQueryParams to have some standard for all providers?
+// Set the QueryParams for the request
+func (p *MeteoBlueProvider) SetQueryParams(coords *plumber.Coordinates) map[string]string {
+	// Get the config to accesss the API Key
```

Comment:
we can add a check to see whether the coords aren't nil ?

Reply:
I thought we could do this when we define the endpoints, before calling fetchData itself